site/src/app/layout.tsx

```diff
@@ -28,7 +42,12 @@ export default async function RootLayout({ children }: { children: React.ReactNo
   const user = await serverTrpc.users.get.query().catch(() => null);
 
   return (
-    <html lang="en" data-theme={user?.theme} suppressHydrationWarning>
+    <html
+      lang="en"
+      data-theme={user?.theme}
+      className={`${roboto.variable} ${openSans.variable}`}
```

Comment:
I believe this should be just Roboto?
```suggestion
      className={roboto.variable}
```